Client/Frontend/Strings.swift

```diff
@@ -36,6 +36,13 @@ extension Strings {
     public static let TopSitesRemoveButtonAccessibilityLabel = NSLocalizedString("TopSites.RemovePage.Button", value: "Remove page - %@", comment: "Button shown in editing mode to remove this site from the top sites panel.")
 }
 
+// Activity Stream.
+extension Strings {
+    public static let ASPageControlButton = NSLocalizedString("AS.PageControl.Button", comment: "The page control button that lets you switch between pages in top sites")
```

Comment:
This doesn't have a value, and in the XLIFF is exposed as 'AS.PageControl.Button', which doesn't make a lot of sense